core/src/main/scala/com/salesforce/op/stages/impl/preparators/SanityCheckerMetadata.scala

```diff
@@ -179,6 +179,9 @@ case class SummaryStatistics
     meta.build()
   }
 
+  private[op] def +(sum: SummaryStatistics): SummaryStatistics = new SummaryStatistics(count, sampleFraction,
```

Comment:
This summation is mainly to allow cases where there are multiple feature vectors, where each one corresponds to its own sanity checker and model selector? In ModelInsights, it looked like it also allowed multiple sanity checkers leading to the same model. Is that possible?

Reply:
yes and yes